charts/streamsets/templates/statefulset.yaml

```diff
@@ -20,14 +20,6 @@ spec:
         app.kubernetes.io/name: {{ include "streamsets.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
-      initContainers:
```

Comment:
@romanki do we need this for onprem ?

Reply:
ah i see in change it did set some permissions, but not sure if it relevant for onprem, we will check it with the next version.